timer/timer.go

```diff
@@ -10,4 +10,6 @@ type Timer interface {
 	StartAt(time.Time) Timer
 	// Finish writes elapsed time for metric
 	Finish(bucket string)
+	// FinishWithLabels writes elapsed time for metric
+	FinishWithLabels(bucket string, labels map[string]string)
```

Comment:
I'd rather changed `Finish` method signature to accept variable list of labels, so that current calls won't be changed, something like this: `Finish(bucket string, label ...Label)`